command/server/config_test_helpers.go

```diff
@@ -456,6 +457,41 @@ func testLoadConfigFile(t *testing.T) {
 	}
 }
 
+func testUnknownFieldValidation(t *testing.T) {
+	config, err := LoadConfigFile("./test-fixtures/config.hcl")
+	if err != nil {
+		t.Fatalf("err: %s", err)
+	}
+
+	expected := []configutil.ConfigError{
+		{
+			Problem:  "unknown field bad_value found in configuration",
+			Position: token.Pos{},
```

Comment:
I don't think the Pos is being checked in the test. If possible, could we see this in action?